.github/workflows/publish-docker.yml

```diff
@@ -73,7 +69,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64 #,linux/arm64
```

Comment:
Is it not possible anymore to build `linux/arm64` after this change?

Reply:
Correct